domain/task/fhir_repository.go

```diff
@@ -102,13 +102,15 @@ func newFHIRTaskFromJSON(data []byte) *fhirTask {
 }
 
 type fhirTaskRepository struct {
-	url     string
+	rest    *resty.Client
 	factory Factory
 }
 
 func NewFHIRTaskRepository(factory Factory, url string) *fhirTaskRepository {
 	return &fhirTaskRepository{
-		url:     url,
+		rest: resty.New().
```

Comment:
OK, this change is gonna hurt my me since I completely refactored this file in my eOverdracht AdvanceNotice composition 😭 . Let leave it for now, but imo such changes ideally come in a separate PR ideally (under-the-hood changes/improvements v.s. new feature).

Although I must admit I'm guilty of this PR smell all too often ;-)